src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> STORAGE = new HashMap<>();
```

Comment:
this is also not a constant, it is in fact constantly being changed by our application
```suggestion
    public static final Map<String, Integer> storage = new HashMap<>();
```